airbyte-integrations/connectors/destination-databricks/src/main/kotlin/io/airbyte/integrations/destination/databricks/jdbc/DatabricksDestinationHandler.kt

```diff
@@ -29,14 +35,12 @@ import java.util.*
 import kotlin.streams.asSequence
 
 class DatabricksDestinationHandler(
+    private val sqlGenerator: DatabricksSqlGenerator,
     private val databaseName: String,
     private val jdbcDatabase: JdbcDatabase,
 ) : DestinationHandler<MinimumDestinationState.Impl> {
 
     private val log = KotlinLogging.logger {}
-    private val abRawId = DatabricksSqlGenerator.AB_RAW_ID
-    private val abExtractedAt = DatabricksSqlGenerator.AB_EXTRACTED_AT
-    private val abMeta = DatabricksSqlGenerator.AB_META
```

Comment:
see DatabricksSqlGenerator for more explanation of what these constants were / why I'm removing them